### DISCUSSION TITLE
Segmentation fault. inside libcrypto.so.1.1

### DISCUSSION THREAD

**Discussion by patryk4815** [dani-garcia/vaultwarden#2404] [Q&A] [answered]
### Segmentation fault
Hi. I have `vaultwarden/server:latest` (arm64) `sha256:152338e09f1fe8389d0fec7baf0aa597e4d6066eff3ff31b7550f0e692e841fb`
Binary is crashing every time when I open web.

<img width="654" alt="Zrzut ekranu 2022-04-5 o 01 58 11" src="https://user-images.githubusercontent.com/3074260/161652803-781d8611-05c4-4f2a-988d-83c353a546e2.png">
<img width="1386" alt="Zrzut ekranu 2022-04-5 o 02 01 16" src="https://user-images.githubusercontent.com/3074260/161652841-2b17225e-08b6-47a2-af68-e37585ec3a96.png">

I just run `apt-get update; apt-get upgrade` inside docker and it fix problem.
Probably there was issue inside `/usr/lib/aarch64-linux-gnu/libcrypto.so.1.1`
Probably it trying to start download icons. After fix i saw this in logs `[2022-04-04 23:54:14.472][vaultwarden::api::icons][INFO] Downloaded icon from https://[redacted]`

### Deployment environment
Apple M1 Max (aarch64)
Docker

### Expected behaviour
Don't crash.

### Actual behaviour
Crashing

**Comment by BlackDex** [comment, marked as answer]
Could you try the current `testing` tagged image and see if that solves the issue? Those images are based upon the bullseye debian version.

**Comment by patryk4815** [comment]
`testing` working fine :)

**Reply by BlackDex** [reply]
Great :)
